### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by IlinaGergova**
## Background

The icons used in the project are mixed - pngs, html entities, symbols that are copy pasted, etc. Some of these icons won't be rendered if the used font doesn't have glif for the symbol.

## Aim

To replace Font awesome with Material and to import all used icons from Material.

### CODE REVIEW

File: src/app/sorting-buttons/sorting-buttons.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="clickable" (click)="emitSort()" id="sorting-buttons">
-  <i *ngIf="hideState === 0" id="bg" class="clickable fa fa-sort"></i>
-  <i *ngIf="hideState === -1" id="asc" class="clickable fa fa-sort-up"></i>
-  <i *ngIf="hideState === 1" id="desc" class="clickable fa fa-sort-down"></i>
+  <span *ngIf="hideState === 0" id="bg" class="material-symbols-outlined">unfold_more</span>
```
Comment (IvoStefanov) [inline src/app/sorting-buttons/sorting-buttons.component.html:2]: Make a bit smaller to match the double arrows size.

File: src/app/users-table/users-table.component.html [hunk 2]
```diff
@@ -56,7 +58,7 @@
                 confirmText="Remove"
                 title="Remove group"
                 message="Remove group?"
-                [iconStyle]="['fa', 'fa-thin', 'fa-times']">
+                [iconStyle]="{ name: 'close', class: 'material-symbols-outlined sm bold' }">
```
Comment (IvoStefanov) [inline src/app/users-table/users-table.component.html:61]: Align button with text.

File: src/app/users-table/users-table.component.html [hunk 3]
```diff
@@ -90,7 +92,7 @@
                 user.email +
                 ' password will be reset. An email with reset instructions will be sent and they won&lsquo;t be able to log in until they set a new password.'
               "
-              [iconStyle]="['fa', 'fa-lg', 'fa-refresh']">
+              [iconStyle]="{ name: 'refresh', class: 'material-symbols-outlined bold' }">
```
Comment (IvoStefanov) [inline src/app/users-table/users-table.component.html:95]: Align action buttons vertically (in center)

**Review by IvoStefanov — CHANGES_REQUESTED** [review]

**Review by joankosev — DISMISSED** [review]
federation token page:
- the edit icon and the X icon after edit, aren't aligned
- on credential creation, copy icon isn't aligned

Share quest modal in genotype browser:
- Copy icon should have other cursor on mouseover

Phenotype tool:
- measurement X icon isn't centered

AGP:
- burger icons are a bit bulky, try making them more small/delicate

**Review by IvoStefanov — APPROVED** [review]

**Pull request merged**